observation_portal/common/configdb.py

```diff
@@ -477,14 +480,14 @@ def get_active_instrument_types(self, location: dict) -> set:
         return instrument_types
 
     def get_guider_for_instrument_name(self, instrument_name):
```

Comment:
Should this method and the next (`is_valid_guider_for_instrument_name`) exclude disabled instruments?

Reply:
I guess so